aboutlibraries-compose/src/androidMain/kotlin/com/mikepenz/aboutlibraries/ui/compose/Libraries.kt

```diff
@@ -82,7 +81,7 @@ fun Libraries(
 ) {
     LazyColumn(modifier, contentPadding = contentPadding) {
         items(libraries) { library ->
-            val openDialog = remember { mutableStateOf(false) }
+            val openDialog = rememberSaveable { mutableStateOf(false) }
```

Comment:
This allows the dialog to be shown again if the device is rotated and the Activity recreated.

Reply:
Thank you @leinardi